### PR TITLE
[DECLUTTER WHY] remove gaining privacy with Wasabi

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch removes the `Gaining Privacy with Wasabi Wallet`, as it is not useful content, and the vast majority of it is already covered in `Bitcoin Privacy`. The great infographic is moved to that chapter.

### CODE REVIEW

File: docs/why-wasabi/BitcoinPrivacy.md [hunk 1]
```diff
@@ -117,6 +119,16 @@ Read more about the privacy concerns of address reuse in the [separate entry](ht
...
+Here one person sends one address to a public forum, like in the bio of a social media network or on a website, and anyone can send bitcoin to this address.
+
+#### Dusting
+
+tb1qg6dzr2yzgyyc7q59w4e4r4yu8ax87zaclh0x36
+
+### Wasabi's Solution
+
+_**Remove used address from GUI**_
+
```
Comment (yahiheb) [inline docs/why-wasabi/BitcoinPrivacy.md]: What is all of this?
This change is in https://github.com/zkSNACKs/WasabiDoc/pull/556 also.

Comment (MaxHillebrand) [reply docs/why-wasabi/BitcoinPrivacy.md]: Yes, I will rebase it and remove conflicts when #556 is merged.

Comment (yahiheb) [reply docs/why-wasabi/BitcoinPrivacy.md]: It is now merged.
My point is don't do the same change in two different PRs to avoid conflicts.

**Comment by MaxHillebrand** [comment]
I messed up with git stuff, and couldn't figure out how to properly rebase this branch - so i created a new one in #561, thus closing this one here.

**Pull request closed without merging**